examples/io/read_impedances_from_ant_neuro.py

```diff
@@ -0,0 +1,69 @@
+"""
+.. _ex-io-ant-impedances:
+
+======================================
+Getting impedances from ANT Neuro .cnt
```

Comment:
For future compat and not needing to add redirects etc. could we rename this "Getting impedances from different EEG formats" or something, and then have a section with antio, then a short section for BrainVision that trivially shows `raw.impedances`? The latter would really just be a placeholder for when we soon show how to do it with `pybv`.